packages/content-type-read-receipt/README.md

```diff
@@ -1 +1,104 @@
-## Coming soon
+# Read receipt content type
+
+This package provides an XMTP content type to support read receipts to messages.
+
+## What’s a read receipt?
+
+A read receipt is a message sent to confirm that a previously sent message has been read by the recipient. With XMTP, read receipts are special messages with the ReadReceipt content type. They contain a timestamp of when the original message was read.
+
+When someone receives a message on an app with read receipts enabled, their XMTP client can send a read receipt when they open that message.
+
+## Why read receipts?
+
+Read receipts give the sender confirmation that the recipient has read their message. This avoids uncertainty on whether a message was seen, without needing to rely on responses.
+
+## Install the package
+
+```bash
+# npm
+npm i @xmtp/content-type-read-receipt
+
+# yarn
+yarn add @xmtp/content-type-read-receipt
+
+# pnpm
+pnpm i @xmtp/content-type-read-receipt
+```
+
+## Create a read receipt
+
+With XMTP, read receipts are represented as objects with the following keys:
+
+- `timestamp`: The timestamp the read receipt was sent
```

Comment:
i think we should explicitly mention that the timestamp should be in ISO 8601 format.

Reply:
great call.